test/common/router/rds_impl_test.cc

```diff
@@ -253,6 +290,36 @@ TEST_F(RdsImplTest, Basic) {
                        ->routeEntry()
                        ->clusterName());
 
+  // Test Admin /routes handler
+  EXPECT_EQ(Http::Code::OK, admin_.runCallback("/routes", data));
+  EXPECT_EQ("route_config_name: foo_route_config\ncluster_name: foo_cluster\nconfig dump:\nname: "
```

Comment:
This test is incredibly difficult to read. Can you redo this using string literals to make it more readable. For stuff above you might consider some kind of utility function that creates a generic response or something.